src/config/summary.rs

```diff
@@ -26,6 +28,9 @@ pub struct Summary {
     // Formatted code differs from existing code (write-mode diff only).
     pub has_diff: bool,
 
+    // What write mode rustfmt was invoked with, if any.
+    pub write_mode: Option<WriteMode>,
```

Comment:
I don't think this needs to be optional - even if there is no explicit write mode, then there is a default.

Reply:
Good point. I'll default it to overwrite?

Reply:
Resolved, defaults to Overwrite.